planetarium/NineChronicles
# Fix Buff skill's tooltip values to 0.

### Description

1. Fix Buff skill's tooltip values to 0.

### How to test

1. Check tables.
2. Check item tooltips

### Related Links

### Required Reviewers

@Namyujeong 

### Screenshot

![image](https://user-images.githubusercontent.com/60082897/133370353-a525ee23-685e-4784-8adb-cf0bd7fb4040.png)
